src/components/create-safe/status/CreationStatus.tsx

```diff
@@ -130,18 +132,24 @@ export const CreationStatus = ({ onClose }: Props) => {
       )}
       {displaySafeLink && (
         <Box mt={3}>
-          <Link href={{ pathname: AppRoutes.safe.home, query: { safe: safeAddress } }} passHref>
-            <Button variant="contained">Open your safe</Button>
-          </Link>
+          <Track {...CREATE_SAFE_EVENTS.GO_TO_SAFE}>
+            <Link href={{ pathname: AppRoutes.safe.home, query: { safe: safeAddress } }} passHref>
+              <Button variant="contained">Open your safe</Button>
+            </Link>
+          </Track>
```

Comment:
I believe we only show this button if the indexing of the safe fails. Should we also track this event once the user is automatically taken to their dashboard after safe creation?

Reply:
"safe" should be capitalized btw.